src/layouts/EditHomepage.jsx

```diff
@@ -1025,20 +1025,19 @@ const EditHomepage = ({ match }) => {
                                 draggableId={`resources-${sectionIndex}-draggable`}
                                 index={sectionIndex}
                                 tag={<Tag variant="subtle">Resources</Tag>}
-                                title={section.resources.title}
+                                title="New resource widget"
```

Comment:
Is this title meant to be fixed?

Reply:
yeap! i initially thought it wasn't gonna be fixed too but i clarified with design - the wording is intended!